plugins/generator-1.16.5/forge-1.16.5/triggers/command_executed.java.ftl

```diff
@@ -5,14 +5,15 @@
 			double i = entity.getPosX();
 			double j = entity.getPosY();
 			double k = entity.getPosZ();
-			String command=event.getParseResults().getReader().getString();
+		    CommandContext<CommandSource> ctx = event.getParseResults().getContext().build(event.getParseResults().getReader().getString());
 			Map<String, Object> dependencies = new HashMap<>();
 			dependencies.put("x",i);
 			dependencies.put("y",j);
 			dependencies.put("z",k);
 			dependencies.put("world",entity.world);
 			dependencies.put("entity",entity);
-			dependencies.put("command",command);
+			dependencies.put("command",event.getParseResults().getReader().getString());
+		    dependencies.put("cmdcontext", ctx);

```

Comment:
Isn't this supposed to be called "arguments" and not "cmdcontext"?

Reply:
I did not test in-game, but I tested inside MCreator which worked... I'm currently re-testing the entire PR, so if I miss a problem, I'm stupid.

Reply:
It is interesting indeed why tests did not catch this